lisa/sut_orchestrator/baremetal/cluster/rackmanager.py

```diff
@@ -0,0 +1,64 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from dataclasses import dataclass
+from typing import Any, Type
+
+from dataclasses_json import dataclass_json
+
+from lisa import RemoteNode, schema
+from lisa.sut_orchestrator.baremetal.cluster.cluster import Cluster, ClusterSchema
+from lisa.util import fields_to_dict
+from lisa.util.logger import get_logger
+from lisa.util.shell import try_connect
+
+
+@dataclass_json()
+@dataclass
+class RackManagerSchema(ClusterSchema):
+    address: str = ""
+    username: str = ""
+    password: str = ""
+    port: int = 22
+
+
+class RackManager(Cluster):
+    def __init__(self, runbook: RackManagerSchema) -> None:
+        super().__init__(runbook)
+        self.rm_runbook: RackManagerSchema = self.runbook
+        self._log = get_logger("rackmanager", self.__class__.__name__)
+
+    @classmethod
+    def type_name(cls) -> str:
+        return "rackmanager"
+
+    @classmethod
+    def type_schema(cls) -> Type[schema.TypedSchema]:
+        return RackManagerSchema
+
+    def deploy(self) -> Any:
+        self._log.debug("deploy client through rackmanager")
```

Comment:
The cluster type should be printed in platform level, so it doesn't need to be replicated in every cluster.